redaxo/src/core/lib/sql/table.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+/**
+ * Class to represent sql tables.
+ *
+ * @author gharlan
+ *
+ * @package redaxo\core
+ */
+class rex_sql_table
+{
+    use rex_instance_pool_trait;
+
+    private $name;
+    private $columns = [];
+
+    public function __construct($name)
+    {
+        $this->name = $name;
+
+        foreach (rex_sql::factory()->showColumns($name) as $column) {
```

Comment:
das würde ich nicht in einem normalen `__construct()` direkt mitmachen.

entweder ne factory methode, die das ganze auch direkt mitmacht (dies aber im namen auch signalisiert) oder eine methode die man separat aufrufen muss